### PR TITLE
Fix build for OpenBSD

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

**Comment by xycaleth** [comment]
Hey David, thanks for the pull request! Can you rename those `_X` variables to just `X` (instead of `__X`)?

Variables names beginning with an underscore followed by a capital letter are reserved for the compiler - same goes for variable names.

File: code/rd-common/mdx_format.h [hunk 1]
```diff
@@ -148,7 +148,7 @@ typedef struct
 	// I'm defining this '<' operator so this struct can be used as an STL <map> key...
 	//
 	#ifdef __cplusplus
-	bool operator < (const mdxaCompQuatBone_t& _X) const {return (memcmp(Comp,_X.Comp,sizeof(Comp))<0);}
+	bool operator < (const mdxaCompQuatBone_t& __X) const {return (memcmp(Comp,__X.Comp,sizeof(Comp))<0);}
```
Comment (ensiform) [inline code/rd-common/mdx_format.h]: What exactly does BSD not like about these variable declarations?

Comment (devnexen) [inline code/rd-common/mdx_format.h]: They are just already defined :-)

**Comment by devnexen** [comment]
@xycaleth ok doing it

**Pull request merged**